app/uk/gov/hmrc/cataloguefrontend/shuttering/model.scala

```diff
@@ -124,14 +141,22 @@ object ShutterStatusValue {
   }
 }
 
-sealed trait ShutterStatus { def value: ShutterStatusValue }
+sealed trait ShutterStatus {
+  def value: ShutterStatusValue
+  def reason: Option[String]
```

Comment:
Whether reason was present or not was previously encoded in the type. Has this been added just to help the UI?
I'd be tempted to add a function for this to `object ShutterStatus`, or even `ShutterEventPage` e.g. `def optReason(ss: ShutterStatus): Option[String]` if required

Reply:
This is purely for the UI, and so I agree that it doesn't really belong in the model.
Have added a ```ShutterEventsView``` object as a helper object for the view to contain this function.  Although a simple function, I didn't want to add this into the template itself.